Network/HTTP/ReverseProxy.hs

```diff
@@ -272,6 +273,13 @@ data WaiProxySettings = WaiProxySettings
     -- Default: no op
     --
     -- @since 0.6.0.1
+    , wpsModifyResponseHeaders :: WAI.Request -> HC.Response () -> HT.ResponseHeaders -> HT.ResponseHeaders
+    -- ^ Allow to override the response headers before the response is returned upstream. Useful for example
+    -- to override overly-strict 'Content-Security-Policy' when the source is known to be trustworthy.
+    --
+    -- Default: no op
+    --
+    -- @since 0.6.0.4
```

Comment:
Sorry, this should be 0.6.1, and should include a ChangeLog entry and cabal file bump.

Reply:
Ah, good catch, I didn't really know what was the protocol here: I'll amend.